src/test/java/core/basesyntax/service/impl/WriterServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.WriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class WriterServiceImplTest {
+    private WriterService writerService;
+    private List<String> originalContent;
+
+    private final String filePath = "src/main/resources/output.csv";
```

Comment:
should be constant

Reply:
Hi! But this is a constant, isn't?  I know that constants usually use uppercase letters with underscores to separate words but when I do it I get this error:   Member name 'FILE_PATH' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9]*$'. [MemberName].  But essentially they are constants. What do you say?

Reply:
make it class-level `private static final String FILE_PATH = "src/main/resources/output.csv";`